src/encoded/tests/conftest.py

```diff
@@ -38,8 +75,26 @@ def app_settings(request, wsgi_server_host_port, conn, DBSession):  # noQA - We
     return settings
 
 
-# INDEXER_NAMESPACE_FOR_TESTING is defined in src/encoded/tests/workbook_fixtures.py
-# (In CGAP it would be defined here, and maybe in the future it will be for Fourfront, too.)
+INDEXER_NAMESPACE_FOR_TESTING = generate_indexer_namespace_for_testing('cgap')
+
+
+@pytest.fixture(scope='session')
+def es_app_settings(wsgi_server_host_port, elasticsearch_server, postgresql_server, aws_auth):
+    settings = make_app_settings_dictionary()
+    settings['create_tables'] = True
+    settings['persona.audiences'] = 'http://%s:%s' % wsgi_server_host_port  # 2-tuple such as: ('localhost', '5000')
```

Comment:
Make sure I didn't make a typo, but maybe:
```suggestion
    host, port = wsgi_server_host_port  # 2-tuple such as: ('localhost', '5000')
    settings['persona.audiences'] = f'http://{host}:{port}'
```